feathr_project/feathr/dtype.py

```diff
@@ -126,4 +149,6 @@ def __init__(self, shape: int, dType: ValueType = ValueType.INT32):
 DOUBLE = DoubleFeatureType()
 STRING = StringFeatureType()
 BYTES = BytesFeatureType()
-FLOAT_VECTOR = FloatVectorFeatureType()
\ No newline at end of file
+FLOAT_VECTOR = FloatVectorFeatureType()
+INT32_VECTOR = Int32VectorFeatureType()
+DOUBLE_VECTOR = DoubleVectorFeatureType()
```

Comment:
add sparse 1d array?
we support int to string, float, double, boolean sparse array. (but can be next PR)